src/components/ContactUs/ContactUs.tsx

```diff
@@ -1,9 +1,94 @@
 import React from 'react'
+import styled from 'styled-components'
+import { mediaQueries } from '../../mediaqueries'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faEnvelope, faPhoneAlt } from '@fortawesome/free-solid-svg-icons';
+import { faFacebook, faInstagram } from '@fortawesome/free-brands-svg-icons';
```

Comment:
tried to install packages and am getting the error below:
Module not found: Can't resolve '@fortawesome/free-brands-svg-icons'

check you didnt install the package globally, cant find it on the package.json dependencies

Reply:
Review pls..?